docs/getting-started-guides/docker.md

```diff
@@ -30,12 +30,12 @@ Documentation for other releases can be found at
 <!-- END STRIP_FOR_RELEASE -->
 
 <!-- END MUNGE: UNVERSIONED_WARNING -->
-Running Kubernetes locally via Docker
```

Comment:
Unfortunately, the diff here is probably what will be committed. If you don't mind, could you change them back?